RibonDAO/core-api
# Prod 730: creation of the fund table to be used instead of the pool

# Description

- creation of the fund table to be used instead of the pool

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Check if donation is working fine
